pymc/func_utils.py

```diff
@@ -11,15 +11,18 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
-import warnings
 
-from collections.abc import Callable
+import warnings
 
-import numpy as np
-import pytensor.tensor as pt
+from sys import modules
 
-from pytensor.gradient import NullTypeGradError
-from scipy import optimize
+try:
+    from preliz.unidimensional import maxent
```

Comment:
this should only happen when someone tries to call `find_constrained_prior` (including the import, to not slowdown pymc import in general)

Also: why do you import modules from sys?